sway-lib-std/src/slice.sw

```diff
@@ -16,4 +28,66 @@ impl<T> &__slice[T] {
         };
         len
     }
+
+    pub fn clone(self) -> &mut [T] {
+        let old_ptr = self.ptr();
+        let len = self.len();
+        let len_in_bytes = __mul(len, __size_of::<T>());
+
+        let new_ptr = asm(len_in_bytes: len_in_bytes, old_ptr: old_ptr) {
+            aloc len_in_bytes;
+            mcp hp old_ptr len_in_bytes;
+            hp: raw_ptr
+        };
+
+        asm(buf: (new_ptr, len)) {
+            buf: &mut [T]
+        }
+    }
+}
+
+pub fn zero_alloc_slice<T>() -> &mut [T] {
+    asm(buf: (0, 0)) {
+        buf: &mut [T]
+    }
+}
+
+pub fn alloc_slice<T>(len: u64) -> &mut [T] {
+    asm(a: 1) {
+        log a a a a;
+    }
```

Comment:
these logs should be removed (and the ones below)?

Reply:
I am not finding this in the last commit, maybe it was something that was already removed.